react/features/always-on-top/AlwaysOnTop.js

```diff
@@ -256,6 +358,10 @@ export default class AlwaysOnTop extends Component<*, State> {
             this._audioAvailabilityListener);
         api.removeListener('videoAvailabilityChanged',
             this._videoAvailabilityListener);
+        api.removeListener('largeVideoChanged',
+            this._largeVideoChangedListener);
+        api.removeListener('displayNameChange',
+            this._displayNameChangedListener);
```

Comment:
Do you want to call removeListener for avatarChanged as well?

Reply:
Yes!